### PR TITLE
Updated Korean translations

### PR DESCRIPTION
**Pull request by ScathachSkadi**
Added new items translations and some UI related translations.

### CODE REVIEW

File: app/src/main/res/values-ko/localized.xml [hunk 1]
```diff
@@ -11,7 +21,7 @@
     <string name="battle_config_support_any">아무거나</string>
 
     <string name="support_imgs_extracted">서포트 이미지가 성공적으로 추출되었습니다</string>
-    <string name="support_imgs_extract_failed">서포트 이미지 추출을 실패했습니다</string>
```
Comment (reconman) [inline app/src/main/res/values-ko/localized.xml:14]: Missing a >

Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml:14]: Thanks

**Review by ScathachSkadi — COMMENTED** [review]
> added

**Comment by item4** [comment]
there are some useless trailing spaces. can you strip these?

**Comment by ScathachSkadi** [comment]
@item4 I actually don't have a lot of coding knowledge to figure out what is wrong exactly... Usually MatthewSachin(Not tagging him just in case he is busy) cleans up the messy bits I made. But I do know what is wrong in general. Is there something that teaches me what parts are wrong so that I can do everything by myself next time?

**Comment by reconman** [comment]
> @item4 I actually don't have a lot of coding knowledge to figure out what is wrong exactly... Usually MatthewSachin(Not tagging him just in case he is busy) cleans up the messy bits I made. But I do know what is wrong in general. Is there something that can teach me what parts are wrong so that I can do everything by myself next time?

He probably means all your added empty lines.

**Comment by ScathachSkadi** [comment]
@reconman Oh. I did that so I can see what I needed to add side by side with Notepad++. If I delete the empty lines will it be still functional?

**Comment by reconman** [comment]
Yes, of course.

**Comment by ScathachSkadi** [comment]
Wow. That is news to me. I'll go do that right now.

**Comment by ScathachSkadi** [comment]
I hope I did it right.

File: app/src/main/res/values-ko/localized.xml [hunk 2]
```diff
@@ -1,5 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+    <string name="tapper_service_description">
+        This accessibility service is used to perform clicks and swipes without root.
```
Comment (reconman) [inline app/src/main/res/values-ko/localized.xml:4]: This also should be translated.

**Review by reconman — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-ko/localized.xml (hunk 2 above)
Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml:4]: Alright.

Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml]: I did it. Thanks for letting me know.

File: app/src/main/res/values-ko/localized.xml [hunk 3]
```diff
@@ -245,10 +227,9 @@
     <string name="turns_count">턴: %d</string>
     <string name="turns_stats">턴: %1$d (최단), %2$d (평균), %3$d (최장)</string>
     <string name="picked_exp_stacks">%d개의 경험치 묶음 획득</string>
-
     <string name="p_mats">재료</string>
     <string name="mats_farmed">%d개의 재료 파밍됨</string>
-
+    <string name="times_rolled">Rolled %d time(s)</string>
```
Comment (item4) [inline app/src/main/res/values-ko/localized.xml:232]: 이거 번역이 가능하지 않을까요?
(I think it is translatable.)

Comment (ScathachSkadi) [reply app/src/main/res/values-ko/localized.xml]: Thank you. It has been fixed now.

**Review by reconman — APPROVED** [review]

**Pull request merged**